catacomb-session does systemd-fu to get the environment setup properly.
However, this is already done by tinydm, so this is not necessary in this case.
Removing it also removes the dependency on systemd.

diff --git a/session/catacomb.desktop b/session/catacomb.desktop
index 65cce48..41d09a7 100644
--- a/session/catacomb.desktop
+++ b/session/catacomb.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Catacomb
 Comment=Wayland Mobile Compositor
-Exec=catacomb-session
+Exec=catacomb
 Terminal=false
 Type=Application

